docs/MigrationGuides.md

```diff
@@ -0,0 +1,15 @@
+# Migration Guides
+
+This project follows [Semantic Versioning](http://semver.org).
+
+Please follow the appropriate guide below when **upgrading to a new major version** of Moya (e.g. 8.0 -> 9.0).
+
+## Upgrade from 8.x to 9.x
+
+- Move the `parameters` and `parameterEncoding` to the `task` computed property by using the case `.requestParameters(parameters:,encoding:)`
```

Comment:
Don't know what I was thinking back then, but I usually use it like that, too. So sorry for that, fixed it via 516764c503bc60b63cf59da49f3798f02ba365de. :)